spec/lss_loader_spec.rb

```diff
@@ -202,9 +202,13 @@ def create_a_single_office
     id
   end
 
-  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
-  def load_from_fixtures(members_csv_filename: "empty", locations_csv_filename: "empty", opening_hours_csv_filename: "empty",
-                         accessibility_info_csv_filename: "empty", volunteer_roles_csv_filename: "empty")
+  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/ParameterLists
```

Comment:
Quite a few cops have been disabled here - is it worth rethinking this method?

Reply:
I have given it some thought on how to cut it down but I think everything else just moves the complexity around and increases duplication in the specs or makes them harder to read rather than have all the complexity involved in setting up the tests and fixtures in one place like this